src/lib/layers/editable-layer.js

```diff
@@ -112,6 +120,18 @@ export default class EditableLayer extends CompositeLayer {
       'pointerup',
       this.state._editableLayerState.pointerHandlers.onPointerUp
     );
+    this.context.gl.canvas.addEventListener(
+      'dblclick',
+      this.state._editableLayerState.pointerHandlers.onDoubleClick
+    );
```

Comment:
@sivako I think you should implement the logic that converts the 2 mouse-downs into a double click instead of using `dblclick`.